hazelcast/test/CMakeLists.txt

```diff
@@ -1,38 +1,50 @@
 cmake_minimum_required(VERSION 3.10)
 
-set(gtest_force_shared_crt ON CACHE BOOL "" FORCE)
-
 if (${CMAKE_SYSTEM_NAME} MATCHES "Linux")
     #required to suppress the warning:
     # `gtest-death-test.cc:1301:24: error: 'dummy' may be used uninitialized [-Werror=maybe-uninitialized]`
     #for googletest source compilation
     set(CMAKE_CXX_FLAGS_DEBUG "${CMAKE_CXX_FLAGS_DEBUG} -Wno-maybe-uninitialized")
 endif ()
 
-find_package(GTest QUIET)
-if (NOT GTest_FOUND)
-    message(STATUS "No existing googletest library installation is found. Will now add googletest as external project by downloading it from github.")
+find_package(GTest 1.10)
```

Comment:
did you test the min. supported version?

Reply:
Just did. The version before 1.10 is 1.8.1, I installed gtest/1.8.1 from Conan and it gave compile errors.